### PR TITLE
Moved models to common entities

### PR DESCRIPTION
**Pull request by anurag94y**

### CODE REVIEW

File: common-entities/src/main/java/com/meeya/Building.java [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
 import java.util.List;
 
 /**
  * @author anurag.y
  * @since 08/10/18.
  */
+@Getter
```
Comment (paras885) [inline common-entities/src/main/java/com/meeya/Building.java]: * Use `@Data` annotation to skip all these imports.
* Add `@Builder` annotation to provide flexibility to build object in a safe way.

**Note:** Apply for each file.

File: core-service/src/main/java/controllers/ParkingSlotController.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package controllers;
```
Comment (paras885) [inline core-service/src/main/java/controllers/ParkingSlotController.java:1]: * Remove this package and create separate repo for core service.
* Also change repo name to **ParkingTicketSystem-CommonEntities**.

File: common-entities/pom.xml [hunk 3]
```diff
@@ -27,7 +27,7 @@
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
```
Comment (paras885) [inline common-entities/pom.xml]: * Please remove unused dependencies.

File: core-service/src/main/java/controllers/ParkingSlotController.java (hunk 2 above)
Comment (anurag94y) [reply core-service/src/main/java/controllers/ParkingSlotController.java:1]: My focus is on making code workable not different repo. 
We can do if it is important at any import that is why I create the different module for the different component.

**Pull request merged**